src/plugins/tracking/util/clickState.js

```diff
@@ -0,0 +1,56 @@
+import JSCookie from 'js-cookie';
+
+/*
+ * This module provides utilities for managing click tracking data in Redux
+ * state an saving that data to a browser cookie
+ */
+
+export const COOKIE_NAME = 'click-track'; // must remain in sync with Meetup Classic implementation
+
+const BrowserCookies = JSCookie.withConverter({
+	read: (value, name) => value,
+	write: (value, name) =>
+		encodeURIComponent(value).replace(
+			/[!'()*]/g,
+			c => `%${c.charCodeAt(0).toString(16)}`
+		),
+});
+
+export const setClickCookie = clickTracking => {
+	const domain = window.location.host.replace(/[^.]+/, ''); // strip leading subdomain, e.g. www or beta2
+	const cookieVal = JSON.stringify(clickTracking);
+	BrowserCookies.set(COOKIE_NAME, cookieVal, { domain });
+};
+
+export const CLICK_TRACK_ACTION = 'CLICK_TRACK';
+export const CLICK_TRACK_CLEAR_ACTION = 'CLICK_TRACK_CLEAR';
+
+export const actions = {
```

Comment:
I noticed later we're renaming `actions` and `reducer` as we import them. 

Is this a naming convention thing, or something that needs to happen?